.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: bash
+
+os: osx
+
+script:
+  # Check style
+  - brew cask style Casks/graalvm-ce.rb
+  # Install cask, extract its version, and update `PATH`
+  - brew cask install --require-sha graalvm/tap/graalvm-ce
+  - VERSION=$(brew cask info graalvm-ce | head -n 1 | awk '{print $2}')
+  - TARGET_DIR="/Library/Java/JavaVirtualMachines/graalvm-ce-${VERSION}"
+  - export PATH="${TARGET_DIR}/Contents/Home/bin:${PATH}"
+  # Print versions of official languages
+  - java -version
```

Comment:
I would just change this to `test ! -z "$(java -version 2>&1 | grep $VERSION)"` just to make sure we have the expected version installed

Reply:
Sure, I added this as a separate check, so that the log also includes `java -version`.